Dockerfile

```diff
@@ -8,7 +8,7 @@
 FROM fcpindi/c-pac:release-v1.6.0
 
 # Note the Maintainer
-MAINTAINER Flywheel <support@flywheel.io>
+LABEL MAINTAINER "Flywheel <support@flywheel.io>"
```

Comment:
Hmmm, this is new to me, so I looked it up.  Apparently it should be LABEL maintainer="support@flywheel.io".  
See https://docs.docker.com/engine/reference/builder/#maintainer-deprecated
and also https://medium.com/@chamilad/lets-make-your-docker-image-better-than-90-of-existing-ones-8b1e5de950d.  Apparently there is a lot more we can add here.